docs/en/observability/monitor-k8s/monitor-k8s-add-integration.asciidoc

```diff
@@ -0,0 +1,380 @@
+*Kubernetes Namespace metrics*:: Monitor namespace active and terminating statuses. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_namespace[`kube-state-metrics` namespace metrics].
+*Kubernetes Node metrics*:: Monitor node health and resource usage. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_node[`kube-state-metrics` node metrics].
+*Kubernetes PersistentVolume metrics*:: Monitor PersistentVolume size, status, and storage configuration. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_persistentvolume[`kube-state-metrics` PersistentVolume metrics].
+*Kubernetes PersistentVolumeClaim metrics*:: Monitor PersistentVolumeClaim phases, classes, and storage requests. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_persistentvolumeclaim[`kube-state-metrics` PersistentVolumeClaim metrics].
+*Kubernetes Pod metrics*:: Monitor pod health and performance. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_pod[`kube-state-metrics` pod metrics].
+*Kubernetes ReplicaSet metrics*:: Monitor ReplicaSets status and the number of replicas in your ReplicaSets. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_replicaset[`kube-state-metrics` ReplicaSet metrics].
+*Kubernetes ResourceQuota metrics*:: Monitor resource limits and current usage. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_resourcequota[`kube-state-metrics` ResourceQuota metrics].
+*Kubernetes Service metrics*:: Monitor service configuration, accessibility, and network integration. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_service[`kube-state-metrics` service metrics].
+*Kubernetes StatefulSet metrics*:: Monitor StatefulSet configuration, status, and scaling. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_statefulset[`kube-state-metrics` StatefulSet metrics].
+*Kubernetes StorageClass metrics*:: Monitor how storage is provisioned and allocated. Learn more at {integrations-docs}/kubernetes/kube-state-metrics#state_storageclass[`kube-state-metrics` StorageClass metrics].
+====
+
+[discrete]
+[[monitor-k8s-kube-state-configure-objects]]
+==== Configure the `kube-state-metrics` options
+
+Provide the following information for each of the `kube-state-metrics` options:
+
+[horizontal]
+*Add Metadata*:: This option adds metadata to events. The metadata is valuable for grouping and exploring related data. This option is on by default.
+*Hosts*:: The address where `kube-state-metrics` is running. Port 8080 is the default.
+*Leader Election*:: When on (default behavior), only the elastic agent which holds the leadership lock will retrieve metrics from the `kube_state_metrics`. This prevents duplicate data in multi-node Kubernetes clusters.
```

Comment:
"holds the leadership lock" is a restrictive clause, so you should use `that` instead of `which`.

```suggestion
*Leader Election*:: When on (default behavior), only the Elastic Agent that holds the leadership lock will retrieve metrics from the `kube_state_metrics`. This prevents duplicate data in multi-node Kubernetes clusters.
```